packages/market-matcher/.env.example

```diff
@@ -0,0 +1,6 @@
+WEB3 = "http://localhost:8545"
+MARKET_CONTRACT_ADDRESS = "0x35E1bA1dEC0bc5A14799563EF8564310846D1438"
+ORIGIN_CONTRACT_ADDRESS = "0xE377ae7241f76aC5Cdb89f5DEbc0dBE82f9c2C03"
+MATCHER_PUB_KEY = "0x585cc5c7829b1fd303ef5c019ed23815a205a59e"
```

Comment:
Do we need to have the public key defined? Can it not be derived from the private key?

Reply:
Yeah, good point.